### PR TITLE
Sentry config

### PR DESCRIPTION
**Pull request by quietbits**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -7,3 +7,47 @@ const nextConfig = {
...
+// Injected content via Sentry wizard below
+
+const { withSentryConfig } = require("@sentry/nextjs");
+
+module.exports = withSentryConfig(module.exports, {
+  // For all available options, see:
+  // https://github.com/getsentry/sentry-webpack-plugin#options
+
+  org: "stellarorg",
+  project: "laboratory",
```
[the remark this answers is not in the document]
Comment (quietbits) [reply next.config.js:20]: No, this is what it's called on Sentry for this project. Thanks for pointing out!

File: next.config.js [hunk 2]
```diff
@@ -7,3 +7,47 @@ const nextConfig = {
...
+  // Automatically annotate React components to show their full name in breadcrumbs and session replay
+  reactComponentAnnotation: {
+    enabled: true,
+  },
+
+  // Uncomment to route browser requests to Sentry through a Next.js rewrite to circumvent ad-blockers.
+  // This can increase your server load as well as your hosting bill.
+  // Note: Check that the configured route will not match with your Next.js middleware, otherwise reporting of client-
+  // side errors will fail.
+  // tunnelRoute: "/monitoring",
```
Comment (jeesunikim) [inline next.config.js:40]: do we not want to enable this b/c of the potential increased hosting bill

**Review by jeesunikim — APPROVED** [review]

File: next.config.js (hunk 2 above)
Comment (quietbits) [reply next.config.js:40]: I think commenting out disables this feature. @xiv, could you please double-check this to make sure it's correct? :pray:

File: sentry.client.config.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+// This file configures the initialization of Sentry on the client.
+// The config you add here will be used whenever a users loads a page in their browser.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+Sentry.init({
+  dsn: "https://478778eb1a5642de97c08f3c3ea33ece@o14203.ingest.us.sentry.io/1773525",
```
Comment (xiv) [inline sentry.client.config.ts:8]: For added security, id recommend moving this to an ENV

File: sentry.edge.config.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+// This file configures the initialization of Sentry for edge features (middleware, edge routes, and so on).
+// The config you add here will be used whenever one of the edge features is loaded.
+// Note that this config is unrelated to the Vercel Edge Runtime and is also required when running locally.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+Sentry.init({
+  dsn: "https://478778eb1a5642de97c08f3c3ea33ece@o14203.ingest.us.sentry.io/1773525",
```
Comment (xiv) [inline sentry.edge.config.ts:9]: We should move this to an ENV for added flexibility and security

File: sentry.server.config.ts [hunk 5]
```diff
@@ -0,0 +1,15 @@
+// This file configures the initialization of Sentry on the server.
+// The config you add here will be used whenever the server handles a request.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+Sentry.init({
+  dsn: "https://478778eb1a5642de97c08f3c3ea33ece@o14203.ingest.us.sentry.io/1773525",
```
Comment (xiv) [inline sentry.server.config.ts:8]: We should move this to an ENV for added flexibility and security

**Review by xiv — CHANGES_REQUESTED** [review]
The Sentry DSN (Data Source Name) is considered sensitive information, but it is not classified as a high-security secret like an API key or database password. So would recommend migrating this to an ENV instead of leaving it hardcoded.

**Pull request closed without merging**